@kiva/kv-components/src/vue/KvUserAvatar.vue

```diff
@@ -144,6 +144,25 @@ export default {
 			isLoading.value = false;
 		};
 
+		const waitForImageToComplete = (img) => {
+			return new Promise((resolve) => {
+				const check = () => {
+					if (img?.complete) {
+						resolve(img);
+					} else {
+						setTimeout(check, 100);
```

Comment:
We might want a max wait time, then fallback to empty circle. I'm wondering if there's a chance for an infinite promise here. Maybe a max wait time of 10s?